.circleci/config.yml

```diff
@@ -0,0 +1,8 @@
+version: 2.1
+
+jobs:
+  build:
+    docker:
+      - image: cimg/base
+    steps:
+      - run: echo "This is an empty config just to keep CircleCI quiet."
```

Comment:
:-D

Reply:
I got a bit tired of it complaining about missing config :P

I'll actually put it in a separate PR but I wasn't sure if it'll even work so I decided to try it out here.

Reply:
#34